modules/vba_documents/app/models/vba_documents/upload_submission.rb

```diff
@@ -3,13 +3,34 @@
 module VBADocuments
   class UploadSubmission < ActiveRecord::Base
     include SetGuid
+    include SentryLogging
+
+    IN_FLIGHT_STATUSES = %w[received processing].freeze
```

Comment:
Is it possible that the record could move from an error state to a success, or would that always require a new submission?

Reply:
Inquiry out to CMAPI team